### DISCUSSION TITLE
Trying to re-do the add of the icon on the editor toolbar (help)

### DISCUSSION THREAD

**Discussion by laurathay** [jcalixte/doc-jit#3] [Q&A] [answered]
Hello there, 

Im creating a vscode plugin but Im having a hard time just adding the icon on the editor toolbar, any chance you could show me how you made it work ? I saw the demo of you plugin doc-jit and it's exactly what I want to achieve ! 

Good job and have a good day :)

**Comment by jcalixte** [comment, marked as answer]
Hi! Sooooo sorry for the delay, if it's still relevant for you, you can check the `package.json` and the "menus" props. You can then associate an icon (used by VS code) and a command you registered in your `src/index.ts`.
